### ISSUE TITLE
Add Functionality to migrate Account from Mastodon easily

### ISSUE THREAD

**Issue by xundeenergie** [friendica/friendica#12477]
### Is the feature request related to a problem? Please describe.

Most users reach the Fediverse via Mastodon. They think it's "Mastodon Network"... they join a lot of other Users and get told, it's easy to migrate to another instance... again mastodon.
If friendica should grow, there is a need for an easy migration-script to change also from mastodon to friendica.

### Describe the feature you'd like

Add functionality for Mastodon-Beginners to migrate to a friendica-account, as easy as in mastodon.

**Comment by kr428** [comment]
Killer feature here would be supporting post migration and/or the ability to "re-import" a Mastodon data export and somehow start all over anew with this. Mastodon is missing that at the moment, it seems quite a requested and very disputed feature over there (and has been so for a while...).

**Comment by tobiasd** [comment]
At the moment contacts exported from a Mastodon account as CSV file, can be imported in Friendica.

**Comment by kr428** [comment]
> At the moment contacts exported from a Mastodon account as CSV file, can be imported in Friendica.

Yes I know, and am regularly exercising that. However, that is something Mastodon's capable of, as well (and to a greater extent than Friendica as it also allows for importing mute/block lists which doesn't seem to be the case in Friendica at the moment). Post migration, on the other hand (as per https://github.com/mastodon/mastodon/issues/12423 ) seems one of the most "controversial" disputes on Mastodon github ever since ... maybe Friendica could be a viable alternative here... ;)

**Comment by loma-one** [comment]
https://github.com/friendica/friendica/issues/6004
